### PR TITLE
Update sftp doc - download action

### PR DESCRIPTION
**Pull request by eeliang**

### CODE REVIEW

File: connectors/sftp/download-file-action.md [hunk 1]
```diff
@@ -5,9 +5,13 @@ date: 2019-10-15 09:00:00 Z
 
 # Download file action
 
-This action downloads a file from your SFTP server. The file contents can be used in subsequent recipe steps. This action cannot be used to download entire folders. 
+This action downloads a file from your SFTP server. This cannot be used to download entire folders. 
 
-The size limit for this action is 70MB. The file will be downloaded in binary mode.
+The file contents will be downloaded in binary mode and can be used in subsequent recipe steps. 
+
+When downloading large files, Workato will split the file contents into chunks and download them across separate requests. The time it takes to process this action depends on the file size and the download speed. This means that larger files will take more time to process, leading to longer job processing time.
```
Comment (eeshansim) [inline connectors/sftp/download-file-action.md]: No need for this line
> ~This means that larger files will take more time to process, leading to longer job processing time.~

File: connectors/sftp/download-file-action.md [hunk 2]
```diff
@@ -5,9 +5,13 @@ date: 2019-10-15 09:00:00 Z
...
+
+There is a timeout limit of **180 seconds** for this action. See here for more information on [timeouts on jobs and actions](/recipes/recipe-job-errors.md#timeouts).
```
Comment (eeshansim) [inline connectors/sftp/download-file-action.md]: Action timeout does not apply for streaming actions. Remove this line.

File: connectors/sftp/download-file-action.md [hunk 3]
```diff
@@ -26,3 +30,4 @@ The output of this action is a **list of entries**. Each entry contains the foll
 | ----------------- | ------------------------- |
 | File content      | The contents of the file. |
 | Detected encoding | The encoding of the file. |
+| Size              | The size of the file.     |
```
Comment (eeshansim) [inline connectors/sftp/download-file-action.md]: The size of the file in bytes.

**Comment by eeshansim** [comment]
@eeliang new implementation introduced a new action for streaming download instead of merging into the old action. Make sure the doc is updated.

**Comment by eeliang** [comment]
1. Added new action.
2. Linked in `SUMMARY.md`, `/sftp.md`, and `/download-file-action.md`.
3. Minify new images
4. Will need to change the helplink

**Pull request merged**